imports/plugins/included/product-detail-simple/client/containers/productDetailContainer.js

```diff
@@ -68,9 +77,9 @@ class ProductDetailContainer extends Component {
         });
       } else {
         productId = currentProduct._id;
-
+        const isDigital = currentProduct.isDigital;
```

Comment:
**Nitpick**
You could use JS destructuring assignment for this if necessary
`const { isDigital }= currentProduct;`